tests/test_cli.py

```diff
@@ -1,33 +1,27 @@
-# -*- coding: utf-8 -*-
-
-from __future__ import unicode_literals, print_function
-
 import argparse
 import logging
 import os
 import platform
 import warnings
 import subprocess
+from configparser import RawConfigParser
```

Comment:
kind of a nit: but let's make import alphabetical, it makes it easier to determine duplicative imports etc.

Reply:
Done! https://github.com/c4urself/bump2version/pull/108/commits/1eb846bb7b6c74955262840273b93b744c21341b